pytest.ini

```diff
@@ -20,20 +20,5 @@ addopts =
     --ignore=docs/utils
 markers =
     only_asyncio: marks tests as only enabled when --reactor=asyncio is passed
-flake8-max-line-length = 119
-flake8-ignore =
-    W503
-
-    # Exclude files that are meant to provide top-level imports
-    # E402: Module level import not at top of file
-    # F401: Module imported but unused
-    scrapy/__init__.py E402
-    scrapy/core/downloader/handlers/http.py F401
-    scrapy/http/__init__.py F401
-    scrapy/linkextractors/__init__.py E402 F401
-    scrapy/selector/__init__.py F401
-    scrapy/spiders/__init__.py E402 F401
-
-    # Issues pending a review:
-    scrapy/utils/url.py F403 F405
-    tests/test_loader.py E741
+filterwarnings=
+    ignore::DeprecationWarning:twisted.web.test.test_webclient
```

Comment:
I think we should not ignore this. We need to fix it instead.

Maybe we can create an issue to deal with this, and add a comment here with a reference to that issue, so that it’s clear that the end goal is to fix the issue, but hide the warning since we are already aware of it and tracking it.

Reply:
File contains two type of entities:
- `SmthResource` which we use
- Deprecated twisted classes with deprecation tests
Importing just `SmthResources` emits warnings. So it is okay to ignore them.

Reply:
I now realize that this will probably only affect tests, since I don’t imagine we use `twisted.web.test` outside tests. Assuming so, I no longer things this needs to be fixed or cannot be hidden.